### PR TITLE
Rebeccayo/weat 207 run pass fail

### PR DESCRIPTION
**Pull request by RebeccaYo**
This is the WEAT-207 bug **Replace zeros in weathervaneResults.csv**

When findmax fails (this does not typically occur and happens only when a maximum number of users is unable to be found), the use of the summary values caused 'uninitialized' warnings, which I disabled. Please let me know if this is not an acceptable solution.

Changelog:
- Add new column "Pass" to the left of the WvUsers column.
- When all AIs pass, Pass=Y. Otherwise, Pass=N.
- For findmax runs, if Pass=N, summary fields contain "NA"
- For non-findmax runs, and passing findmax runs, summary fields contain sum or average across all AIs
- Remove the column labeled "workload"
- Remove the column labeled "workloadProfile".
- Rename the column labeled "users-total" to WvUsers.

### CODE REVIEW

File: runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm [hunk 1]
```diff
@@ -1928,26 +1928,45 @@ sub getWorkloadStatsSummary {
 	my $overallAvgRT = 0;
 
 	my $numPassingAi = 0;
+	my $numTotalAi = 0;
 	my $appInstancesRef = $self->workload->appInstancesRef;
+	$numTotalAi = scalar @$appInstancesRef;
```
Comment (bhoflich) [inline runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm]: I think all other parts of the code used $# with +1 to get number of elements, although scalar should work also.

File: runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm [hunk 2]
```diff
@@ -1928,26 +1928,45 @@ sub getWorkloadStatsSummary {
...
+			
+	if ($numPassingAi == $numTotalAi) { # If all AIs pass, the run passes
+		$csvRef->{"Pass"}				= "Y";
 	} else {
-		$csvRef->{"overallAvgRT-total"} = 0;		
+		$csvRef->{"Pass"}				= "N";
 	}
+
+	# If this is a failed findMax run, print "NA" for summary fields
+	if ( ($csvRef->{"Pass"} eq "N") && ($self->workload->appInstancesRef->[0]->getParamValue('loadPathType') eq "findmax") ) {
```
Comment (bhoflich) [inline runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm]: Should it check for syncedfindmax also?

Comment (hrosenbe) [reply runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm]: Yes.  You need to check for syncedfindmax here as well.  I don't think that loadPathType existed in the code you were using originally.

Comment (RebeccaYo) [reply runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm]: Thank you for the catch. This has been fixed in commit ebc4db1.

**Pull request merged**